src/__tests__/required-props-for-components.ts

```diff
@@ -133,6 +133,9 @@ const defaultProps: Record<string, Record<string, any>> = {
   'area-chart': {
     series: [],
   },
+  flashbar: {
```

Comment:
This is necessary to feed the component with dummy a `items` array in unit tests which started failing after I added logic which relies on the prop existing. Since the prop is mandatory, it's better to update this here than to make the component code permissive.